src/server/mcp.ts

```diff
@@ -1165,13 +1167,21 @@ export class ResourceTemplate {
  * - `content` if the tool does not have an outputSchema
```

Comment:
nit: could you update this comment? (cf. https://modelcontextprotocol.io/specification/2025-06-18/server/tools#structured-content)

```
- `content`: if an outputSchema is defined, content *SHOULD* have the serialized JSON structuredContent in a text content for backwards compatibility
```